test/TEST-16-ISSUE-518/avocado.conf

```diff
@@ -0,0 +1,2 @@
+[datadir.paths]
+logs_dir = /tmp/job-results
```

Comment:
BTW https://anonscm.debian.org/cgit/pkg-systemd/systemd.git/tree/debian/tests/upstream can't run this test of course. So, ubuntu-CI will fail